pydoctor/test/test_astbuilder.py

```diff
@@ -1210,16 +1210,16 @@ def type2html(obj: model.Documentable) -> str:
     assert type2html(d) == '<code>str</code>'
     e = C.contents['e']
     assert e.docstring == """fifth"""
-    assert type2html(e) == '<code>List[C]</code>'
+    assert type2html(e).replace('<wbr></wbr>', '').replace('<wbr>\n</wbr>', '') == '<code>List[C]</code>'
```

Comment:
Since `type2html()` is a test helper function, you could put the `<wbr>` removal there.